policy/templates.go

```diff
@@ -36,6 +37,83 @@ func (r *RepoPolicy) GenTemplate(bundle string) error {
 	return r.renderTemplates(bundlePath)
 }
 
+// GenTerraformPolicyTemplate generates the terraform policy file
+// from the given template file.
+func GenTerraformPolicyTemplate(fpath string, fileName string, policy Policies) error {
+
+	opFile := filepath.Join("policy/terraform/github", fileName)
+	op, err := os.Create(opFile)
+	if err != nil {
+		return err
+	}
+	t := template.Must(template.
+		New(filepath.Base(fileName)).
+		Funcs(sprig.FuncMap()).
+		Option("missingkey=error").
+		ParseFiles(filepath.Join(fpath, fileName)),
+	)
+	log.Debug().Str("tmpl", fileName).Str("output", opFile).Msg("rendering terraform tmpl")
+	// Set current timestamp if not set already
+	// if r.Timestamp == "" {
+	// 	r.SetTimestamp(time.Time{})
+	// }
+	err = t.Execute(op, policy)
+	if err != nil {
+		return err
+	}
+	log.Debug().Msg("templates rendered successfully")
+	return nil
+}
+
+func GenTerraformPolicyTemplate2(fPath string, policy Policies) error {
+
+	return fs.WalkDir(templates, fPath, func(path string, d fs.DirEntry, errWalk error) error {
```

Comment:
This construction feels awkward, and is probably a re-implementation of walking logic that has been implemented for the releng bundles. Please unify the implementation of walking a template bundle across all bundles.

Reply:
Please lets discuss about this one, walking over wont work for this implementation since we need to target specific file names.

Reply:
The goal is to not care about the bundle contents. We should not be letting filenames leak into this.